docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:20.04
```

Comment:
If we want to follow the main repo architecture we should move the docker file from this folder to the root one.

Reply:
ok . If sure can move to root folder but as separate folder , it looks more organized and further we can also trigger docker build from any folder as there is no such limitation.